dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -25,7 +31,12 @@ object TankGenerator {
   private def mkAssets(count: Int = settings.assets.count): List[String] = {
     println(s"Generating $count assets... ")
     val assets = (1 to count).map(_ => mkAsset()).toList
-    waitForHeightArise()
+    val asset  = assets.get(new Random().nextInt(assets.length - 1))
+
+    do {
+      waitForHeightArise()
+    } while (services.matcher.getTradableBalance(new AssetPair(asset, "WAVES"), issuer.getAddress()).getOrDefault(asset, 0L) <= 0)
```

Comment:
You will create AssetPair every time you enter the loop
So, extract asset pair

Reply:
ok